webui/react/src/components/InfoBox.tsx

```diff
@@ -5,25 +5,40 @@ import css from './InfoBox.module.scss';
 export interface InfoRow {
   content?: React.ReactNode;
   label: string;
+  onClick?: () => void,
+}
+
+export enum InfoboxStyle {
+  Default,
+  Boxed,
 }
 
 interface Props {
   rows: InfoRow[];
+  style?: InfoboxStyle,
 }
 
-export const renderRow = ({ label, content }: InfoRow): React.ReactNode => {
+export const renderRow = ({ content, label, onClick }: InfoRow): React.ReactNode => {
   if (!content) return null;
+
+  const classes = [ css.info ];
+  if (onClick) classes.push(css.infoClickable);
+
   return (
-    <div className={css.info} key={label}>
+    <div className={classes.join(' ')} key={label} onClick={onClick}>
       <div className={css.label}>{label}</div>
       <div className={css.content}>{content}</div>
     </div>
   );
 };
 
-const InfoBox: React.FC<Props> = ({ rows }: Props) => {
+const InfoBox: React.FC<Props> = ({ rows, style = InfoboxStyle.Default }: Props) => {
+  const classes = [];
+  if (style === InfoboxStyle.Boxed) classes.push(css.boxed);
+  if (style === InfoboxStyle.Default) classes.push(css.default);
+
   return (
-    <div className={css.base}>
+    <div className={classes.join(' ')}>
```

Comment:
**suggestion:** we have a `Grid` component that can handle the layout of the InfoBoxes.

Reply:
Does `Grid` allow to easily collapse boxes (Cols) on multiple rows?

Reply:
it supports wrapping, so on the mobile view it'll wrap like what you have (if that's what you're asking)

Reply:
Cluster and Dashboard have some OverviewStats + Grid examples

Reply:
My bad I didn't look into if there was something re-usable for this, so thanks.

Reply:
I should've mentioned it up front. Thanks for refactoring!